### PR TITLE
Fix randomness of #newTravisID

### LINKED ISSUE #529: Pharo tests fail on Windows
smalltalkCI currently ignores failures for Pharo (and Moose) on Windows. [If it doesn't ignore them](https://github.com/hpi-swa/smalltalkCI/commit/c2b07922a693fa3f01519138f814cfb896763c62), we get a [couple of test failures](https://github.com/hpi-swa/smalltalkCI/runs/2663356580?check_suite_focus=true):

image

If someone with access to a Windows machine could have look, that'd be great. There may be an issue with [UUIDs](https://github.com/hpi-swa/smalltalkCI/blob/c2b07922a693fa3f01519138f814cfb896763c62/repository/SmalltalkCI-Core.package/SmalltalkCI.class/class/newTravisID.st)?

### PR DESCRIPTION
**Pull request by theseion**
The first couple of bytes of the UUID in Pharo are clock based. Testing `#newTravisID` can thus lead to clashes if the clock hasn't advanced enough yet (appears to be a problem on Windows only, possibly due to clock resolution).

The last couple bytes of a UUID are fixed (version number etc.).

So... I did use the "beautiful" implementation, using `Random` directly, however, it's a lot more code than should be necessary, because the `Random` class is quite different between Squeak and Pharo.

Fixes #529

### CODE REVIEW

**Comment by theseion** [comment]
The failure appears to be an issue with Coveralls, the tests are green.

File: repository/SmalltalkCI-Core.package/SmalltalkCI.class/class/newTravisID.st [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+	| bytes random |
+	bytes := ByteArray new: 2.
+	random := Random new.
+	1
+		to: 2
+		do: [ :index |
+			bytes
+				at: index
+				put: (random nextInt: 255) ].
+	^ bytes hex
```
Comment (fniephaus) [inline repository/SmalltalkCI-Core.package/SmalltalkCI.class/class/newTravisID.st:13]: Does this work in Pharo? Or why did you decide to use `#to:do:`?

```suggestion
	| r |
	r := Random new.
	^ (ByteArray with: r nextInt with: r nextInt) hex
```

**Comment by fniephaus** [comment]
Should/can we revert https://github.com/hpi-swa/smalltalkCI/commit/c2b07922a693fa3f01519138f814cfb896763c62?

**Comment by theseion** [comment]
> Should/can we revert [c2b0792](https://github.com/hpi-swa/smalltalkCI/commit/c2b07922a693fa3f01519138f814cfb896763c62)?

Not sure, what you mean... I think that the Windows tests should be fine now, so I wouldn't ignore failures.

**Comment by fniephaus** [comment]
By reverting https://github.com/hpi-swa/smalltalkCI/commit/c2b07922a693fa3f01519138f814cfb896763c62, we no longer ignore failures on Windows...that's what I meant :)

**Comment by fniephaus** [comment]
Cheery-picked via https://github.com/hpi-swa/smalltalkCI/commit/46e8d5fb8e9239760f9e180c3c2aad507b38be29.

**Pull request merged**